packages/rfw/test/runtime_test.dart

```diff
@@ -347,6 +347,17 @@ void main() {
     expect(tester.widget<ColoredBox>(find.byType(ColoredBox)).color, const Color(0x00000002));
   });
 
+  testWidgets('Runtime', (WidgetTester tester) async {
+    final Runtime runtime = Runtime()
+          ..update(const LibraryName(<String>['core']), createCoreWidgets());
+    for (final MapEntry<LibraryName, WidgetLibrary>(key: LibraryName libraryName, value: WidgetLibrary widgetLibrary) in runtime.libraries.entries) {
```

Comment:
Given that we know it has a single entry should we use something like map.entries.single and then make the assertions?

Reply:
Done.